Build failed: too few arguments to function ‘SDL_CreateWindowAndRenderer’

Just cloned the SDL repo, built and installed fine, but this one fails:

```
/home/rena/projects/games/SDL_image/examples/showanim.c: In function ‘main’:
/home/rena/projects/games/SDL_image/examples/showanim.c:86:50: warning: passing argument 4 of ‘SDL_CreateWindowAndRenderer’ makes integer from pointer without a cast [-Wint-conversion]
   86 |     if (SDL_CreateWindowAndRenderer(0, 0, flags, &window, &renderer) < 0) {
      |                                                  ^~~~~~~
      |                                                  |
      |                                                  SDL_Window **
In file included from /usr/local/include/SDL3/SDL.h:69,
                 from /home/rena/projects/games/SDL_image/examples/showanim.c:22:
/usr/local/include/SDL3/SDL_render.h:214:115: note: expected ‘SDL_WindowFlags’ {aka ‘unsigned int’} but argument is of type ‘SDL_Window **’
  214 | LSPEC int SDLCALL SDL_CreateWindowAndRenderer(const char *title, int width, int height, SDL_WindowFlags window_flags, SDL_Window **window, SDL_Renderer **renderer);
      |                                                                                         ~~~~~~~~~~~~~~~~^~~~~~~~~~~~

/home/rena/projects/games/SDL_image/examples/showanim.c:86:59: warning: passing argument 5 of ‘SDL_CreateWindowAndRenderer’ from incompatible pointer type [-Wincompatible-pointer-types]
   86 |     if (SDL_CreateWindowAndRenderer(0, 0, flags, &window, &renderer) < 0) {
      |                                                           ^~~~~~~~~
      |                                                           |
      |                                                           SDL_Renderer **
/usr/local/include/SDL3/SDL_render.h:214:142: note: expected ‘SDL_Window **’ but argument is of type ‘SDL_Renderer **’
  214 | eWindowAndRenderer(const char *title, int width, int height, SDL_WindowFlags window_flags, SDL_Window **window, SDL_Renderer **renderer);
      |                                                                                            ~~~~~~~~~~~~~^~~~~~

/home/rena/projects/games/SDL_image/examples/showanim.c:86:9: error: too few arguments to function ‘SDL_CreateWindowAndRenderer’
   86 |     if (SDL_CreateWindowAndRenderer(0, 0, flags, &window, &renderer) < 0) {
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/include/SDL3/SDL_render.h:214:29: note: declared here
  214 | extern DECLSPEC int SDLCALL SDL_CreateWindowAndRenderer(const char *title, int width, int height, SDL_WindowFlags window_flags, SDL_Window **window, SDL_Renderer **renderer);
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [CMakeFiles/showanim.dir/build.make:76: CMakeFiles/showanim.dir/examples/showanim.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:141: CMakeFiles/showanim.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

--- a/examples/showanim.c
+++ b/examples/showanim.c
@@ -83,7 +83,7 @@ int main(int argc, char *argv[])
         return(2);
     }
 
-    if (SDL_CreateWindowAndRenderer(0, 0, flags, &window, &renderer) < 0) {
+    if (SDL_CreateWindowAndRenderer("animation demo", 0, 0, flags, &window, &renderer) < 0) {
         SDL_Log("SDL_CreateWindowAndRenderer() failed: %s\n", SDL_GetError());
         return(2);
     }

